trie/node.go

```diff
@@ -275,14 +276,22 @@ func shouldStopIfContextDoneBlockingIfBusy(ctx context.Context, idleProvider Idl
 	}
 }
 
-func treatCommitSnapshotError(err error, hash []byte, missingNodesChan chan []byte) {
+func treatCommitSnapshotError(err error, hash []byte, missingNodesChan chan []byte) (bool, error) {
```

Comment:
maybe use named return values? 
not very clear what the bool return value signifies.

Reply:
Done.